jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -312,14 +312,23 @@
           max-total: 12
           trigger-phrase: 'e2e\s+test'
           cmd: |
-            export KUBE_SKIP_PUSH_GCS=y
-            export KUBE_RUN_FROM_OUTPUT=y
-            export KUBE_FASTBUILD=true
-            ./hack/jenkins/build.sh
             if [[ "${{ghprbTargetBranch:-}}" == "release-1.0" || "${{ghprbTargetBranch:-}}" == "release-1.1" ]]; then
               echo "PR GKE job disabled for legacy branches."
               exit
             fi
+
+            export KUBE_SKIP_PUSH_GCS=n
+            export KUBE_GCS_RELEASE_BUCKET=kubernetes-release-pull
+            export KUBE_RUN_FROM_OUTPUT=y
+            export KUBE_FASTBUILD=true
+
+            ./hack/jenkins/build.sh
+
+            version=$(source build/util.sh && echo $(kube::release::semantic_version))
+            gsutil -m cp -r "gs://kubernetes-release-pull/ci/${{version}}" "gs://kubernetes-release-dev/ci/${{version}}"
```

Comment:
Right, forgot about `latest-build.txt`.

Since this is a bucket -> bucket copy, suggest using `gsutil rsync`, which should be significantly faster. I think the incantation you want is

`gsutil -m rsync -r "gs://kubernetes-release-pull/ci/${{version}}" "gs://kubernetes-release-dev/ci/${{version}}"`

but test it out.